lib/bobbin-sys/src/heap.rs

```diff
@@ -20,7 +20,7 @@
 use ::core::{mem, ptr, slice, str, fmt};
 
 extern "C" {
-    static mut _sheap: u32;
+    static mut __sheap: u32;
```

Comment:
@jcsoo I have changed it according latest changes in cortex-m-rt library. This PR shouldn't be merged unless other boards will be also updated to latest cortex-m-rt.

Reply:
I have updated rest of boards to cortex-m-rt